src/opm/output/eclipse/WriteInit.cpp

```diff
@@ -559,6 +558,63 @@ namespace {
 
         initFile.write("TRANNNC", singlePrecision(tran));
     }
+
+    // output aquifer cell and aquifer connection information for numerical aquifers
+    void writeNumericalAquifers(const Opm::NumericalAquifers& num_aquifers,
+                                const ::Opm::EclipseGrid&          grid,
+                                ::Opm::EclIO::OutputStream::Init&  initFile)
+    {
+        std::vector<int> aquifern(grid.getNumActive(), 0);
+        // aquifer cells
+        const auto& aquifer_cells = num_aquifers.allAquiferCells();
+        for ([[maybe_unused]] const auto& [cell_idx, cell] : aquifer_cells) {
+            const size_t active_index = grid.activeIndex(cell->global_index);
+            aquifern[active_index] -= std::exp2(cell->aquifer_id - 1);
```

Comment:
Different from the connections, an aquifer cell should not belong to two aquifers. Here should use the assignment. Although it should not change things in the results.